corehq/apps/sms/filters.py

```diff
@@ -86,6 +86,13 @@ def filter_context(self):
         return context
 
 
+class PhoneNumberOrEmailFilter(BaseSimpleFilter):
```

Comment:
Shouldn't this replace `PhoneNumberReportFilter` below?

Reply:
The `PhoneNumberReportFilter` is actually used by https://github.com/dimagi/commcare-hq/blob/8fc3a0395275a147fbaaa6df3adb268007dbe92b/corehq/apps/reports/standard/sms.py#L1102, not by the Messaging History. Ours uses the `PhoneNumberFilter` above, and I didn't replace that because `RequiredPhoneNumberFilter` depends on it and that is used somewhere else.